wait_for.sh

```diff
@@ -161,12 +162,23 @@ get_service_state() {
 # in a 'kubectl describe' job output.
 get_job_state() {
     get_job_state_name="$1"
-    get_job_state_output=$(kubectl describe jobs "$get_job_state_name" $KUBECTL_ARGS 2>&1)
+    i=0
+
```

Comment:
Given the customer problem that we observed, it may be priudent to build in a retry delay.

Maybe leverage $WAIT_TIME for this?